bazel/integration/test_runner/file_system_utils.ts

```diff
@@ -26,7 +26,12 @@ export async function isExecutable(filePath: string): Promise<boolean> {
  * the actual case-exact path for the current platform would be: `C:\Users\<..>`.
  */
 export async function getCaseExactRealpath(filePath: string): Promise<string> {
-  return trueCasePath(filePath);
+  // If the exact case path is not found, we return the file path as provided. There are some cases
+  // it seems where certain symbols (potentially ~) cause inconsistencies in true-case-paths
+  // ability to find the path. As this project is abandoned, we do not expect any sort of fix in the
```

Comment:
where is the project abandoned? Sorry, I still don't have enough context here.

Reply:
https://github.com/Profiscience/true-case-path

No activity in the repo since August of 2020, outstanding issues and PRs in it.

Reply:
Addressed by your PR.